src/components/ValidateCodeActionModal/ValidateCodeForm/BaseValidateCodeForm.tsx

```diff
@@ -64,6 +64,9 @@ type ValidateCodeFormProps = {
     /** Function to clear error of the form */
     clearError: () => void;
 
+    /** Whether to show the verify button (hidden in private domain onboarding)  */
```

Comment:
"private domain onboarding" is kind of ambiguous. Can we remove that part? It is obvious what this prop does.

Reply:
Handled.